tests/network/vmi_networking.go

```diff
@@ -243,36 +243,58 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		})
 	})
 
-	Context("VirtualMachineInstance with custom interface model", func() {
+	Context("VirtualMachineInstance with custom and default interface models", func() {
+		const (
+			secondaryNetName = "secondary-net"
+			nadName          = "simple-bridge"
+		)
+		BeforeEach(func() {
+			const linuxBridgeNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"plugins\": [{\"type\": \"bridge\", \"bridge\": \"%s\"}]}"}}`
+			ns := testsuite.GetTestNamespace(nil)
+			Expect(libnet.CreateNetworkAttachmentDefinition(nadName, ns,
+				fmt.Sprintf(linuxBridgeNAD, nadName, ns, nadName, secondaryNetName),
+			)).To(Succeed())
+		})
 		It("[test_id:1770]should expose the right device type to the guest", func() {
 			By("checking the device vendor in /sys/class")
 			// Create a machine with e1000 interface model
 			// Use alpine because cirros dhcp client starts prematurely before link is ready
 			masqIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
 			masqIface.Model = "e1000"
 			masqIface.PciAddress = "0000:02:01.0"
+
+			bridgeIface := libvmi.InterfaceDeviceWithBridgeBinding(secondaryNetName)
+			bridgeIface.PciAddress = "0000:03:00.0"
+
+			bridgeNetwork := libvmi.MultusNetwork(secondaryNetName, nadName)
```

Comment:
Since this variable is only used for the VMI spec definition, please consider specifying the network inline:
```go
libvmifact.NewAlpine(
...
libvmi.WithNetwork(libvmi.MultusNetwork(secondaryNetName, nadName)),
...
```